src/components/QueryBuilder.vue

```diff
@@ -222,4 +227,9 @@ a {
 .querybuilder__run {
 	margin-block-start: $dimension-layout-medium;
 }
+
+.querybuilder__add-condition {
+	float: inline-end;
```

Comment:
do we want to use flex here? in that case we need a flex wrapper only for this element

Reply:
seems like it has terrible browser support.
https://caniuse.com/?search=inline-end

Reply:
I think our postcss is supposed to take care of that, but somehow this is not working. I.e. it is broken in chrome.

Reply:
On inspecting it further, that part of postcss seems to correctly replace it with `float: right` and `float: left`, but we are not picking it up because there is still some confusion about the `dir` attribute. I'll look.